owners/OWNERS.example.yaml

```diff
@@ -58,17 +59,40 @@
 - "*.{js,css}": frontend
 - "package.{,lock.}json": somebodyelse
 
+# Prepending a user or team name with a `?` will prevent this owner from being
```

Comment:
Line 16: For better visibility due to IDE coloring, would it be possible to uncomment the line (`@dontdothis`)? Or will that break parsing?